src/test/java/com/thealgorithms/sorts/ExchangeSortTest.java

```diff
@@ -0,0 +1,41 @@
+package com.thealgorithms.sorts;
+// added by @555vedant//
+
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ExchangeSortTest {
```

Comment:
What do you think about using [`SortingAlgorithmTest`](https://github.com/TheAlgorithms/Java/blob/a216cb8a59ad04b3cadcc156e69dea41f3d1b465/src/test/java/com/thealgorithms/sorts/SortingAlgorithmTest.java#L9) here? Please have a look at [`MergeSortTest`](https://github.com/TheAlgorithms/Java/blob/a216cb8a59ad04b3cadcc156e69dea41f3d1b465/src/test/java/com/thealgorithms/sorts/MergeSortTest.java#L3).